solutions/devsprint-cecilia-saraiva-1/GitHubApp/Screens/List/ListView.swift

```diff
@@ -85,7 +85,7 @@ extension ListView: UITableViewDataSource {
     public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
 
         let cell = tableView.dequeueReusableCell(withIdentifier: self.listViewCellIdentifier)!
-        cell.textLabel?.text = self.listItems[indexPath.row]
+        cell.textLabel?.text = self.listItems[indexPath.row].name
```

Comment:
[suggestion] since [indexPath.row] can crash, it's a good practice to check first and only then configure the text. So maybe let's add a if and do that.